tests/integration/test_ytsaurus/test_tables.py

```diff
@@ -73,82 +73,267 @@ def test_yt_simple_table_function(started_cluster):
+            "[{name = a; type_v3 = {type_name=tuple; elements=[{type=double;};{type=float;};];};};]",
+            "[0.1,1.0]",
+            "Tuple(Float64, Float32)",
+            "(0.1,1)",
+            id="tuple",
+        ),
+        pytest.param(
+            "[{name = a; type_v3 = {type_name=dict; key=int64; value={type_name=optional;item=string;};};};]",
+            "[{\"42\": \"value\"}]",
+            "Map(Int64, Nullable(String))",
+            "{42: 'value'}",
+            id="dict",
+        ),
+        pytest.param(
+            "[{name = a; type_v3 = {type_name=variant; elements=[{type=int32;};{type=string;};];};};]",
+            "[0, 42]",
+            "Variant(Int32, String)",
+            "[0,42]",
+            id="variant",
+        ),
+        pytest.param(
+            "[{name = a; type_v3 = {type_name=variant; elements=[{type=int32;};{type=string;};];};};]",
+            "[1, \"value\"]",
+            "Variant(Int32, String)",
+            "[1,\"value\"]",
+            id="variant",
+        ),
+        pytest.param(
+            "[{name = a; type_v3 = {type_name=tagged; tag=\"image\\svg\"; item=double;};};]",
+            "0.1",
+            "Float64",
+            "0.1",
+            id="tagged",
+        ),
+        pytest.param(
+            "[{name = a; type = any;};]",
             "[1, 2]",
             "Array(Int32)",
             "[1,2]",
             id="Array_simple",
         ),
         pytest.param(
-            "any",
+            "[{name = a; type = any;};]",
             "[[1,1],[1,1]]",
             "Array(Array(Int32))",
             "[[1,1],[1,1]]",
             id="Array_complex",
         ),
         pytest.param(
-            "any",
+            "[{name = a; type = any;};]",
             '{"a":"hello", "38 parrots":[38]}',
             "String",
             '{"a":"hello","38 parrots":[38]}',
             id="Dict",
         ),
     ],
 )
-def test_ytsaurus_types(
+def test_ytsaurus_composite_types(
     started_cluster, yt_data_type, yt_data, ch_column_type, ch_data_expected
 ):
     yt = YTsaurusCLI(started_cluster, instance, "ytsaurus_backend1", 80)
     table_path = "//tmp/table"
     column_name = "a"
     yt_data_json = f'{{"{column_name}":{yt_data}}}\n'
-
-    yt.create_table(table_path, yt_data_json, schema={column_name: yt_data_type})
-
+    create_command = f"yt create table {table_path} --attributes " + " \'{schema = " + yt_data_type + "}\'"
+    print(create_command)
+    yt.exec(create_command)
+    if len(yt_data) > 0:
+        yt.write_table(table_path, yt_data_json)
     instance.query(
         f"CREATE TABLE yt_test(a {ch_column_type}) ENGINE=YTsaurus('{YT_URI}', '{table_path}', '{YT_DEFAULT_TOKEN}')"
     )
-    assert instance.query("SELECT a FROM yt_test") == f"{ch_data_expected}\n"
+    if len(yt_data) > 0:
+        yt.write_table(table_path, yt_data_json)
+        assert instance.query("SELECT a FROM yt_test") == f"{ch_data_expected}\n"
     instance.query("DROP TABLE yt_test")
```

Comment:
```suggestion
    instance.query("DETACH TABLE yt_test")
    instance.query("ATTACH TABLE yt_test")
    
    instance.query("DROP TABLE yt_test")
```
It occurred to me that it would be a good idea to check this(simulating a restart).